src/query/api/v1/middleware/rewrite.go

```diff
@@ -101,6 +101,10 @@ func rewriteRangeDuration(
 			r.Form.Del(endParam)
 		}
 
+		if r.Method == "GET" {
```

Comment:
should this be before the above check?

Reply:
Nah, we still want to purge from `r.Form` the modifications we made solely for the sake of time parsing.